src/win32structs.h

```diff
@@ -83,15 +83,21 @@
 #define MWT_LEFTMULTIPLY	2
 #define MWT_RIGHTMULTIPLY	3
 
+typedef guint32 ARGB;
+typedef float REAL;
+
+#if defined(_WIN32)
+#define NOGDI
+#include <Windows.h>
+#else
+
 typedef int LANGID;
```

Comment:
Indeed. I'm not sure what the impact of the bug was or how it would manifest itself (maybe binary compatability?), but I fixed it anyway